cardano-node/src/Cardano/Node/Tracing/Era/HardFork.hs

```diff
@@ -346,3 +346,21 @@ instance All (LogFormatting `Compose` WrapForgeStateUpdateError) xs => LogFormat
 
 instance LogFormatting (ForgeStateUpdateError blk) => LogFormatting (WrapForgeStateUpdateError blk) where
     forMachine dtal = forMachine dtal . unwrapForgeStateUpdateError
+
+--
+-- instances for HardForkSelectView
+--
+
+instance All (LogFormatting `Compose` WrapSelectView) xs => LogFormatting (HardForkSelectView xs) where
+    -- elide BlockNo as it is already contained in every per-era SelectView
+    forMachine dtal = forMachine dtal . dropBlockNo . getHardForkSelectView
```

Comment:
Not sure I understand, do you mean something like this: 

```
forMachine = const $ forMachine DMinimal
``` 
?

Reply:
I'm not sure. It would be the best if @jutaro could confirm that.

Reply:
I think
```haskell
forMachine = const $ forMachine DMinimal
```
will just loop at runtime as it is calling `forMachine` instantiated at the same type, or am I missing sth?

Reply:
Sounds like it's safer to remove this commit and leave the original version (which presumably has been tested),  and then improve the code after the release, instead of risking this blowing up at runtime (unless someone knowledgeable of this can  clarify). 
On the other hand, merging of the PR (and thus of the release) is blocked until the requested changes have been approved.

Reply:
I was thinking like: `forMachine dtal = forMachine DMinimal . getHardForkSelectView`, but then you have to look that in the concrete instances of the different selectViews, and implement the forMachine code, so that it doesn't print the blockNr., if the detail level is minimal.

Reply:
I removed the commit and added a TODO with the explanation from this thread.  Resolving the conversation.